docs/dev/topology-over-raft.md

```diff
@@ -22,15 +22,81 @@ Nodes in state `left` may still appear as tablet replicas in host_id-based repli
 State transition diagram for nodes:
 ```mermaid
 stateDiagram-v2
-    none --> bootstrapping|replacing
+    none --> bootstrapping: join
+    none --> replacing: replace
+    state bootstrapping {
+        bs_join_group0 : join_group0
+        bs_left_token_ring : left_token_ring
+        bs_commit_cdc_generation : commit_cdc_generation
+        bs_write_both_read_old : write_both_read_old
+        bs_write_both_read_new : write_both_read_new
+        [*] --> bs_join_group0
+        bs_join_group0 --> bs_left_token_ring: rollback
+        bs_join_group0 --> [*]: rejected
+        bs_join_group0 --> bs_commit_cdc_generation
+        bs_commit_cdc_generation --> bs_write_both_read_old
+        bs_write_both_read_old --> bs_write_both_read_new: streaming completed
+        bs_write_both_read_new --> [*]
+        bs_left_token_ring --> [*]
+    }
+    state replacing {
+        rp_join_group0 : join_group0
+        rp_left_token_ring : left_token_ring
+        rp_tablet_draining : tablet_draining 
+        rp_write_both_read_old : write_both_read_old
+        rp_write_both_read_new : write_both_read_new
+        [*] --> rp_join_group0
+        rp_join_group0 --> rp_left_token_ring: rollback
```

Comment:
Here as well (and same for all operations) rollback can be entered from write_both_read_old